xml_inscriptions/transcribed/UT.Provo.BYU.HBLL.G.2011.07.01.xml

```diff
@@ -0,0 +1,171 @@
+                            <supportDesc ana="#metal.gold">
+                                <support>
+                                    <p> Golden amulet</p>
+                                    <dimensions type="surface" unit="cm">
+                                        <height>3</height>
+                                        <width>5.5</width>
+                                        <depth/>
+                                    </dimensions>
+                                </support>
+                                <condition ana="#complete">
+                                    <p/>
+                                </condition>
+                            </supportDesc>
+                            <!--  *************************************  <layoutDesc>  ********************************* -->
+                            <layoutDesc>
+                                <layout columns="4" writtenLines="0">
+                                    <p/>
+                                </layout>
+                            </layoutDesc>
+                        </objectDesc>
+
+                        <!--  *************************************  <handDesc>  ********************************* -->
+
+                        <handDesc hands="1">
+                            <handNote ana="#impressed.inscribed"> </handNote>
+                        </handDesc>
+
+                        <!--  *************************************  <decoDesc>  *********************************     -->
+                        <decoDesc>
+                            <decoNote>
+                                <p/>
+                            </decoNote>
+                        </decoDesc>
+                    </physDesc>
+
+                    <!--  *************************************  <history>  *********************************  -->
+
+                    <history>
+
+                        <summary>
+                            <rs/>  <!-- content can be "fake" or blank -->
+                        </summary>
+
+                        <origin><date notBefore="-0001" notAfter="0300">first - third centuries
+                                C.E.(?)</date>
+                            <placeName ref="asia.near_east">Palestine(?)</placeName>
+                        </origin>
+                        <provenance type="transferred" subtype="acquired" when-custom="1983-1986"> <p>From the <orgName>Royal Athena Gallery in New York</orgName> by a <persName role="acquier">Mr.
+                                    David H. Swingler</persName> of <orgName>Oxnard,
+                                    California</orgName> between <date notBefore="1982"
+                                    notAfter="1986">1983-1985</date></p></provenance>
+                        <provenance type="transferred" subtype="donated" when="2006"><p>y <persName role="donor">Mr. Swingler</persName> to the <orgName>Harold B. Lee Library at Brigham
```

Comment:
Something seems to have gone missing here at the beginning of your `<p>` here? Also, I'd mark this up as `<orgName>Harold B. Lee Library</orgName> at <orgName>Brigham Young University</orgName>, <placeName>Utah</placeName>`